code/modules/mob/living/carbon/xenomorph/castes/shrike/abilities_shrike.dm

```diff
@@ -0,0 +1,150 @@
+	"<span class='xenowarning'>You call forth the larvas to rise from their slumber!</span>")
+	notify_ghosts("\The <b>[S]</b> is calling for the burrowed larvas to wake up!", source = S, NOTIFY_JOIN_AS_LARVA)
+
+	addtimer(CALLBACK(S, /mob/living/carbon/xenomorph/shrike.proc/calling_larvas_end), CALLING_BURROWED_DURATION)
+
+	add_cooldown()
+
+
+/mob/living/carbon/xenomorph/shrike/proc/calling_larvas_end()
+    calling_larvas = FALSE
+
+
+// ***************************************
+// *********** Psychic Fling
+// ***************************************
+/datum/action/xeno_action/activable/psychic_fling
+	name = "Psychic Fling"
+	action_icon_state = "fling"
+	mechanics_text = "Knock a target flying up to 3 tiles. Ranged ability."
+	cooldown_timer = 12 SECONDS
+	plasma_cost = 100
+	keybind_signal = COMSIG_XENOABILITY_PSYCHIC_FLING
+
+
+/datum/action/xeno_action/activable/psychic_fling/on_cooldown_finish()
+	to_chat(owner, "<span class='notice'>You gather enough mental strength to fling something again.</span>")
+	return ..()
+
+
+/datum/action/xeno_action/activable/psychic_fling/can_use_ability(atom/A, silent = FALSE, override_flags)
+	. = ..()
+	if(!.)
+		return FALSE
+	if(!A)
+		return FALSE
+	var/mob/living/carbon/xenomorph/shrike/S = owner
+	var/max_dist = (S == S.hive.living_xeno_ruler) ? 6 : 3 //We can fling targets further away if we are the ruler.
+	if(get_dist(S, A) > max_dist)
+		return FALSE
+	if(!ishuman(A))
+		return FALSE
+	var/mob/living/carbon/human/H = A
+	if(H.stat == DEAD)
+		return FALSE
+
+
+/datum/action/xeno_action/activable/psychic_fling/use_ability(atom/A)
+	var/mob/living/carbon/xenomorph/shrike/S = owner
+	var/mob/living/carbon/human/H = A
+	round_statistics.psychic_flings++
+
+	S.visible_message("<span class='xenowarning'>A strange and violent psychic aura is suddenly emitted from \the [S]!</span>", \
+	"<span class='xenowarning'>You violently fling [H] with the power of your mind!</span>")
+	H.visible_message("<span class='xenowarning'>[H] is violently flung to the side by an unseen force!</span>", \
+	"<span class='xenowarning'>You are violently flung to the side by an unseen force!</span>")
+	playsound(S,'sound/effects/magic.ogg', 75, 1)
+	playsound(H,'sound/weapons/alien_claw_block.ogg', 75, 1)
+
+	succeed_activate()
+	H.apply_effects(1, 2) 	// Stun
+	shake_camera(H, 2, 1)
+
+	var/facing = get_dir(S, H)
+	var/fling_distance = 3
+	var/turf/T = H.loc
+	var/turf/temp
+
+	for(var/x in 1 to fling_distance)
+		temp = get_step(T, facing)
+		if(!temp)
+			break
+		T = temp
+	H.throw_at(T, fling_distance, 1, S, TRUE)
+
+	add_cooldown()
+
+
+// ***************************************
+// *********** Psychic Choke
+// ***************************************
+/datum/action/xeno_action/activable/psychic_choke
+	name = "Psychic Choke"
+	action_icon_state = "screech"
+	mechanics_text = "Stun and start choking a target. Ranged ability."
+	cooldown_timer = 30 SECONDS

```

Comment:
Feels a little high for what is basically a ranged pounce. In effect this is weaker than old neurotoxin, at a much higher cooldown. The abilities function nearly identical, with the exception that this one can fudge over marines that decide running off alone is a good idea. 

I'd knock it down to 20, to be more compareable with pounce.

Reply:
This is very different from a ranged pounce:
1) You are not stunned by doing it.
2) The target can't get freed unless if one of you are moved, or you get otherwise attacked.

Similarly, this has nothing to do with old neurotoxin:
1) This has no miss chance.
2) It's not a short stun.
3) You can't stunlock and move a target if you are a fast spitter.

This serves for an entirely different purpose than either abilities, and it's very low in lethallity if you are alone, unless if you have all the time in the world.

I don't want this to be the new lunge, so we'll see how it goes before buffing it more.